src/SFA.DAS.SelfService.Infrastructure/Client/ReleaseClient.cs

```diff
@@ -127,10 +142,23 @@ public static VstsRelease CreateRelease(ReleaseHttpClient releaseClient, int rel
             {
                 Id = release.Id,
                 ReleaseName = release.Name,
-                ReleaseDefininitionId = release.ReleaseDefinitionReference.Id
+                ReleaseDefininitionId = release.ReleaseDefinitionReference.Id,
+                ReleaseEnvironments = release.Environments.Select(x => new VstsEnvironment { DefinitionId = x.DefinitionEnvironmentId, EnvironmentReleaseId = x.Id, Name = x.Name }).ToList()
             };
 
             return vstsRelease;
         }
+
+        public void StartEnvironmentDeployment(VstsRelease vstsRelease, int releaseEnvironmentId)
+        {
+            ReleaseHttpClient releaseClient = _vssConnection.GetClient<ReleaseHttpClient>();
```

Comment:
is this supposed to be vss or vsts?

Reply:
Good spot, check out line 18
private readonly VssConnection _vssConnection;

The object from the package is VssConnection :( It stands for VisualStudio.Services
using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi;